index.js

```diff
@@ -12,3 +12,25 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+async function fetchCharacters() {
+  try {
+    const response = await fetch("https://rickandmortyapi.com/api/character");
+    if (response.ok) {
+      const data = await response.json();
+      // console.log(data);
+      data.results.forEach((character) => {
+        const characterCard = document.createElement("li");
```

Comment:
I think Line 23 and 24 would be replaced by:
createCharacterfunction(character)

Reply:
ahhh i see